cs.c

```diff
@@ -584,9 +591,14 @@ size_t CAPSTONE_API cs_disasm(csh ud, const uint8_t *buffer, size_t size, uint64
 			insn_cache->address = offset;
 			insn_cache->size = (uint16_t)skipdata_bytes;
 			memcpy(insn_cache->bytes, buffer, skipdata_bytes);
+#ifdef CAPSTONE_DIET
+ 			insn_cache->mnemonic[0] = '\0';
```

Comment:
if i remember correctly, we already set mnemonic & op_str to 0 size somewhere. or sometimes you still see mnemonic & op_str with values?